### PR TITLE
Resolve build container errors

### PR DESCRIPTION
**Pull request by Emilgardis**
Fixes 

```
#21 22.27 Get:2 http://ftp.ports.debian.org/debian-ports unreleased InRelease [50.3 kB]
#21 22.34 Err:1 http://ftp.ports.debian.org/debian-ports unstable InRelease
#21 22.34   The following signatures couldn't be verified because the public key is not available: NO_PUBKEY B523E5F3FC4E5F2C NO_PUBKEY 8D69674688B6CB36
#21 22.42 Err:2 http://ftp.ports.debian.org/debian-ports unreleased InRelease
#21 22.42   The following signatures couldn't be verified because the public key is not available: NO_PUBKEY B523E5F3FC4E5F2C NO_PUBKEY 8D69674688B6CB36
#21 22.43 Reading package lists...
#21 22.44 W: GPG error: http://ftp.ports.debian.org/debian-ports unstable InRelease: The following signatures couldn't be verified because the public key is not available: NO_PUBKEY B523E5F3FC4E5F2C NO_PUBKEY 8D69674688B6CB36
#21 22.44 E: The repository 'http://ftp.ports.debian.org/debian-ports unstable InRelease' is not signed.
#21 22.44 W: GPG error: http://ftp.ports.debian.org/debian-ports unreleased InRelease: The following signatures couldn't be verified because the public key is not available: NO_PUBKEY B523E5F3FC4E5F2C NO_PUBKEY 8D69674688B6CB36
#21 22.44 E: The repository 'http://ftp.ports.debian.org/debian-ports unreleased InRelease' is not signed.
```

### CODE REVIEW

**Comment by Emilgardis** [comment]
bors try --target powerpc*

**Comment by Emilgardis** [comment]
powerpc has moved to use crosstool on rust ci, might have to do the same here, jessie is getting old

**Comment by Emilgardis** [comment]
bors try

File: docker/linux-image.sh [hunk 1]
```diff
@@ -398,6 +399,9 @@ EOF
     if [ -f /etc/dpkg/dpkg.cfg.d/multiarch.bak ]; then
         mv /etc/dpkg/dpkg.cfg.d/multiarch.bak /etc/dpkg/dpkg.cfg.d/multiarch
     fi
+    if [ -f /etc/apt/apt.conf.d/10-nocheckvalid ]; then
+        rm /etc/apt/apt.conf.d/10-nocheckvalid
+    fi
```
Comment (Emilgardis) [inline docker/linux-image.sh:404]: this makes sense to do, but it's not super user friendly

**Comment by Emilgardis** [comment]
gcc download_prerequisites is taking a while :/ timing out solaris

**Comment by Emilgardis** [comment]
bors try

does this work now?

**Comment by Emilgardis** [comment]
the workflow is refusing to show the logs... https://github.com/cross-rs/cross/actions/runs/5348635709/jobs/9698850417

**Comment by Emilgardis** [comment]
bors try

**Comment by aughey** [comment]
This is excellent work.  I have a need right now to use this.  I am looking forward to seeing this merged in and images built.

**Comment by Emilgardis** [comment]
going to merge this as-is, there's more work to do but this has been waiting for a while

**Pull request merged**